Vector.c

```diff
@@ -69,8 +69,8 @@ static bool Vector_isConsistent(const Vector* this) {
    return true;
 }
 
-bool Vector_countEquals(const Vector* this, unsigned int expectedCount) {
-   unsigned int n = 0;
+bool Vector_countEquals(const Vector* this, size_t expectedCount) {
+   size_t n = 0;
    for (int i = 0; i < this->items; i++) {
```

Comment:
Shouldn't this be updated to `size_t` here too (I know this would add a cast to `this->items`).